content/introduction.md

```diff
@@ -6,7 +6,7 @@ Under the hood, Nest makes use of robust HTTP Server frameworks like [Express](h
 
 Nest provides a level of abstraction _above_ these common NodeJS frameworks (Express/Fastify), but also exposes their APIs directly to the developer. This allows developers the freedom to use the myriad of third-party modules which are available for the underlying platform.
 
-> Note: Remember Nest (or NestJS) is an extension/abstraction above NodeJS itself - although similary named to libraries like (Next.js/Nuxt.js) which are server-side rendering abstractions/wrappers around React & Vue respectively - **NestJS is intended to be a NodeJS API framework** and isn't similar or comparable to Nextjs & Nuxtjs.
```

Comment:
```suggestion
> Note: Remember Nest (or NestJS) is an extension/abstraction above NodeJS itself - although similarly named to libraries like (Next.js/Nuxt.js) which are server-side rendering abstractions/wrappers around React & Vue respectively - **NestJS is intended to be a NodeJS API framework** and isn't similar or comparable to Next.js & Nuxt.js.
```

Some guys are really passionate about correct naming. Let's correctly adress them, so we do not piss anyone off 😹